src/subscription_manager/managercli.py

```diff
@@ -3228,7 +3233,8 @@ def _do_command(self):
                 if ex.code == 400:
                     # black listed overrides specified.
                     # Print message and return a less severe code.
-                    system_exit(1, ex)
+                    mapped_message: str = ExceptionMapper().get_message(ex)
+                    system_exit(1, mapped_message)
```

Comment:
This is missing in main, though. Can you please create a PR for it, and then add it as separate commit?

Reply:
For 1.28 (e.g. here) split into separate commit, for main opened as PR #2767